flink-core/src/main/java/org/apache/flink/configuration/SecurityOptions.java

```diff
@@ -168,7 +168,7 @@ public class SecurityOptions {
                     .withDeprecatedKeys("yarn.security.kerberos.additionalFileSystems")
                     .withDescription(
                             "A comma-separated list of Kerberos-secured Hadoop filesystems Flink is going to access. For example, "
-                                    + "security.kerberos.access.hadoopFileSystems=hdfs://namenode2:9002,hdfs://namenode3:9003. "
+                                    + "security.kerberos.access.hadoopFileSystems=hdfs://namenode2:9002;hdfs://namenode3:9003. "
```

Comment:
comma-separated -> semicolon-separated

Reply:
@JunRuiLee 
Thank you for your reply 
I also changed it!